[new-product] Add greenlight

Rudloff opened the pull request:
Followup to https://github.com/endoflife-date/endoflife.date/pull/6091

I could not find an exact latest release for the 1.x branch, the Git repository does not have any tags for this branch.

Rudloff committed:
Create greenlight.md

marcwrobel committed:
formatting

marcwrobel commented on products/greenlight.md:
```diff
@@ -0,0 +1,23 @@
+iconSlug: bigbluebutton
+permalink: /greenlight
+releasePolicyLink: https://github.com/bigbluebutton/greenlight/security
+
```
```suggestion

auto:
  methods:
  -   git: https://github.com/bigbluebutton/greenlight.git

```

Latest release update can be automated.

marcwrobel commented on products/greenlight.md:
```diff
@@ -0,0 +1,23 @@
+releasePolicyLink: https://github.com/bigbluebutton/greenlight/security
+
+releases:
+-   releaseCycle: "3"
```
All release are missing `releaseDate` and `latestReleaseDate` fields.

marcwrobel commented on products/greenlight.md:
```diff
@@ -0,0 +1,23 @@
+category: server-app
+iconSlug: bigbluebutton
+permalink: /greenlight
+releasePolicyLink: https://github.com/bigbluebutton/greenlight/security
```
```suggestion
```

Let's document this directly in the description with a more informative sentence.

Rudloff replied:
I'm not sure about this one.
If the default sentence is not great, wouldn't it make more sense to improve it instead of rewriting it in every product page?

marcwrobel replied:
It's not great only in a few cases, such as when there is only a table listing all versions with their status (such as https://github.com/bigbluebutton/greenlight/security). The vast majority on products on endoflife.date have more than that, and that default sentence is adapted to those cases.

In this particular case I find it more informative to have this:

```
Greenlight supported versions are documented on <https://github.com/bigbluebutton/greenlight/security>.
```

Instead of this:

```
More information is available on the [Greenlight website](https://github.com/bigbluebutton/greenlight/security).
```

Rudloff replied:
Fair enough, you have a point.

marcwrobel commented on products/greenlight.md:
```diff
@@ -0,0 +1,23 @@
+
+---
+
+> [Greenlight](https://docs.bigbluebutton.org/greenlight/v3/install/) is an open-source web user interface for BigBlueButton.
```
```suggestion
> [Greenlight](https://docs.bigbluebutton.org/greenlight/v3/install/) is an open-source web user interface for BigBlueButton.

Greenlight supported versions are documented on <https://github.com/bigbluebutton/greenlight/security>.
```

More explicit than the default sentence IMO, WDYT ?

marcwrobel reviewed (changes requested):
Thanks @Rudloff, here is my review.

Rudloff committed:
Apply suggestions from code review

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

Rudloff committed:
Update greenlight.md

Rudloff committed:
Update greenlight.md

Remove 1.x branch
Could not find release notes and it is very old so probably not useful to have it

Rudloff committed:
Apply suggestions from code review

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

marcwrobel committed:
add changelogtemplate

marcwrobel committed:
formatting + set eol for 2

marcwrobel committed:
Fix auto

marcwrobel reviewed (approved):
Thanks @Rudloff.

I took the liberty to update a few things, see PR commits. Feel free to amend through a follow up PR if needed.
